bundle/internal/schema/annotations_openapi_overrides.yml

```diff
@@ -216,32 +216,15 @@ github.com/databricks/cli/bundle/config/resources.ModelServingEndpoint:
     "description": |-
       PLACEHOLDER
 github.com/databricks/cli/bundle/config/resources.Pipeline:
-  "_":
-    "markdown_description": |-
-      The pipeline resource allows you to create <DLT> [pipelines](/api/workspace/pipelines/create). For information about pipelines, see [_](/delta-live-tables/index.md). For a tutorial that uses the <DABS> template to create a pipeline, see [_](/dev-tools/bundles/pipelines-tutorial.md).
```

Comment:
Why are we losing the documentation for the top-level DLT pipeline resource? Can we avoid this? This seems like it'll flow downstream to the DABs docs generated by the schema.

Reply:
cc @ilyakuz-db as it seems related to change from `[]any` to `any` if I'm not mistaken